provider/src/test/java/com/synopsys/integration/alert/provider/blackduck/processor/detail/RuleViolationClearedNotificationContentDetailExtractorTest.java

```diff
@@ -0,0 +1,53 @@
+package com.synopsys.integration.alert.provider.blackduck.processor.detail;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Optional;
+
+import org.junit.jupiter.api.Test;
+
+import com.google.gson.Gson;
+import com.synopsys.integration.alert.common.rest.model.AlertNotificationModel;
+import com.synopsys.integration.alert.processor.api.detail.DetailedNotificationContent;
+import com.synopsys.integration.alert.test.common.TestResourceUtils;
+import com.synopsys.integration.blackduck.api.manual.component.PolicyInfo;
+import com.synopsys.integration.blackduck.api.manual.component.RuleViolationClearedNotificationContent;
+import com.synopsys.integration.blackduck.api.manual.enumeration.NotificationType;
+
+public class RuleViolationClearedNotificationContentDetailExtractorTest {
+    public static final String NOTIFICATION_JSON_PATH = "json/ruleViolationClearedNotification.json";
+
+    private final Gson gson = new Gson();
+
+    @Test
+    public void extractDetailedContentTest() throws IOException {
```

Comment:
I tried to extract this functionality for reuse, but it bogged down the test and killed readability.